### PR TITLE
made navbar sticky and transparent

### PR DESCRIPTION
**Pull request by Adarshvarghese**
i made the navbar sticky and transparent .but when i made navbar sticky, in .header-bar class, justify-content became inactive.

### CODE REVIEW

File: public/styles.css [hunk 1]
```diff
@@ -87,10 +87,21 @@
 
 .header-bar {
   display: flex;
-  justify-content: space-between;
+  justify-content: baseline;
```
Comment (ajayesivan) [inline public/styles.css:90]: Why did you change it `baseline`?

File: public/styles.css [hunk 2]
```diff
@@ -87,10 +87,21 @@
...
   font-family: "Rubik";
   font-style: normal;
   padding-bottom: 20px;
+  padding-inline: 30PX;
+  position: fixed;
```
Comment (ajayesivan) [inline public/styles.css:95]: When using `position: fixed` you also need to use properties like `left`, `top` etc. to position the elements sproperly.

Please invest some time to learn about CSS `position` properties & `flexbox`

**Review by ajayesivan — CHANGES_REQUESTED** [review]
Also, use `backdrop-filter: blur` to add background blur to the header.

File: public/styles.css (hunk 1 above)
Comment (Adarshvarghese) [reply public/styles.css:90]: sorry,I tried other properties to see the changes and forgot to change it back.

File: public/styles.css (hunk 2 above)
Comment (Adarshvarghese) [reply public/styles.css:95]: I will go through  the topics that you mantioned.

File: public/index.html [hunk 3]
```diff
@@ -49,14 +49,16 @@
 
   <section class="body">
     <div class="header-bar">
-      <img src="images/icon.svg" class="icon">
+      <div><img src="images/icon.svg" class="icon"></div>
```
Comment (ajayesivan) [inline public/index.html:52]: Why do we need a `<div>` here?

File: public/index.html [hunk 4]
```diff
@@ -49,14 +49,16 @@
...
       <nav class="nav-bar">
         <a class="nav-button" href="#home1">home</a>
         <a class="nav-button" href="#howitworks2">how it works</a>
         <a class="nav-button" href="#faq3">faq</a>
         <button class="menu" onclick="menuIconClick()"><img src="images/menu.svg"></button>
       </nav>
     </div>
+
```
Comment (ajayesivan) [inline public/index.html:60]: Remove this spacing.

File: public/styles.css [hunk 5]
```diff
@@ -90,9 +90,15 @@
   justify-content: space-between;
   font-family: "Rubik";
   font-style: normal;
-  padding-bottom: 20px;
-}
+  width: 100%;
```
Comment (ajayesivan) [inline public/styles.css:93]: Add a max-width of 1300px for the header content.

File: public/styles.css (hunk 2 above)
Comment (ajayesivan) [reply public/styles.css:94]: Great! Once you have a good idea about the position property please add `left` and `top` properties to position the header correctly and update this PR.

File: public/styles.css [hunk 6]
```diff
@@ -90,9 +90,15 @@
...
+  position: fixed;
+  background-color: rgba(255, 255, 255, 0.5);
+  margin-top: -20px;
```
Comment (ajayesivan) [inline public/styles.css:96]: Remove this negative margin.

File: public/styles.css [hunk 7]
```diff
@@ -90,9 +90,15 @@
...
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
```
Comment (ajayesivan) [inline public/styles.css:98]: Change blur to `10px`

Note: In the Figma design you can check the **Inspect** tab to get the correct values for most of the CSS properties.

File: public/styles.css [hunk 8]
```diff
@@ -110,6 +116,10 @@
   display: flex;
   gap: 30px;
   align-items: center;
+  position: fixed;
+  right: 100px;
+  margin-top: 45px;
+
```
Comment (ajayesivan) [inline public/styles.css:122]: We don't need this.

**Review by ajayesivan — CHANGES_REQUESTED** [review]
Good progress @Adarshvarghese. We need to fix a few more things before we can merge this PR.

File: public/index.html (hunk 3 above)
Comment (Adarshvarghese) [reply public/index.html:52]: i tried to move the icon using the div but forgot to undo it.

**Comment by ajayesivan** [comment]
@Adarshvarghese Still the design and result look different. Please try to make this look similar to the design. If you are having trouble, please let us know.

**Comment by Adarshvarghese** [comment]
i am facing some trouble at this point.please guide me.

**Comment by Adarshvarghese** [comment]
design view
above is the design screenshot 
scr1
this is the view in my device when i commented out left property in header-bar

**Comment by ajayesivan** [comment]
Screenshot 2023-06-07 at 12 57 35

This is how it looks in my system.

@Adarshvarghese please refer this: https://codepen.io/Ajay-Sivan/pen/rNQNJyY

**Comment by Adarshvarghese** [comment]
sure i will refer the link .
the screen shots i attached  was after i commented  out the left property in header-bar .and i did not push that change at that time.that is why i mentioned it in the above messsage. I have pushed the changes now.  
please check again and confirm

**Comment by ajayesivan** [comment]
@Adarshvarghese Desktop view looks good. Please check and update the mobile view too - https://www.figma.com/file/rd8upWhFjqFRaX4hKTTOh3/leeway?type=design&node-id=550-1309&t=AosDZ8ax8oOd8otX-4

Also, make sure the header looks fine on all screen sizes. You can compare the PR preview build with https://leeway.zympl.xyz

**Comment by Adarshvarghese** [comment]
i have made the remaining changes according to design of mobile view. 
img1
img2

above images are the views in my device after the changes made.

File: public/app.js [hunk 9]
```diff
@@ -23,11 +23,23 @@ function expand(elementId) {
 function menuIconClick() {
     const menuClick = document.getElementById('menuId');
     menuClick.style.display = "block";
+    const logo= document.getElementById('logo');
+    logo.style.display="none"
+    const  hamurgerMenu= document.getElementById('menuIcon');
+    hamurgerMenu.style.display="none";
+    const headerBar = document.querySelector('.header-bar');
+    headerBar.style.display="none"
```
Comment (ajayesivan) [inline public/app.js:31]: @Adarshvarghese Why do we need to change js in a style-related PR?

**Comment by ajayesivan** [comment]
Your branch | current production
-|-
zympl-leeway--pr84-sticky-navbar-h5xhi34g web app_(Pixel 5) | leeway zympl xyz_(Pixel 5)


- leeway logo is missing
- menu icon is on the left. it should be on the right.

**Comment by ajayesivan** [comment]
@Adarshvarghese Please re-request review once you have updated the PR. Please also resolve the conflicts. If you have any questions let me know.

**Pull request closed without merging**